### PR TITLE
v3 Update

### LINKED ISSUE #500: Bump version, test release build and merge to master
- Increment app version
- Test release build manually
- Create PR to merge dev to master

@mejariamol Can you take this up?

### PR DESCRIPTION
**Pull request by mejariamol**
- app name is made configurable by setting env variable app_name

fixes #500

### CODE REVIEW

**Comment by mejariamol** [comment]
@iamareebjamal please review.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Open Event Organizer</string>
+    <string name="app_name">eventyay organizer</string>
```
Comment (niranjan94) [inline app/src/main/res/values/strings.xml]: @mejariamol @iamareebjamal please do not hard code the app name. Make it configurable during the build process.

Comment (mejariamol) [inline app/src/main/res/values/strings.xml]: creating a separate issue for that. @niranjan94 
Actually, I am not sure how to do it. You mean when building from command line, the app name will be passed as a parameter in the command. right?

Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: @niranjan94 Please suggest a solution. We'll have to add app name in either build.gradle or strings.xml (refrerred from AndroidManifest.xml)

Configuration may be done via environment variable and switched to default string if it is not present. Is this OK?

Comment (niranjan94) [inline app/src/main/res/values/strings.xml]: Yep. Environment variables sounds good.

Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: @mejariamol Please make the required changes

**Comment by iamareebjamal** [comment]
@mejariamol Please also update coverage label to development branch

**Comment by mejariamol** [comment]
@iamareebjamal please review.

**Comment by mejariamol** [comment]
will update the doc in a separate PR.

File: app/build.gradle [hunk 2]
```diff
@@ -4,6 +4,8 @@ apply plugin: 'io.sentry.android.gradle'
 apply plugin: 'com.noveogroup.android.check'
 apply from: './jacoco.gradle'
 
+def app_name = System.getenv('app_name') == null ? "eventyay organizer" : System.getenv('app_name')
```
Comment (iamareebjamal) [inline app/build.gradle]: Calling `getenv` twice

**Comment by mejariamol** [comment]
@iamareebjamal please review.

File: app/build.gradle [hunk 3]
```diff
@@ -4,6 +4,8 @@ apply plugin: 'io.sentry.android.gradle'
 apply plugin: 'com.noveogroup.android.check'
 apply from: './jacoco.gradle'
 
+def app_name = System.getenv('app_name') ?: "eventyay organizer"
```
Comment (iamareebjamal) [inline app/build.gradle]: Environment variables should generally be upper cased. Dunno how I missed that in initial review

Comment (mejariamol) [reply app/build.gradle]: yeah. Initially, I tried APP_NAME but that is always returning `Gradle` no matter what is set. you may suggest another name. but app_name or APP_NAME is convenient.

Comment (iamareebjamal) [reply app/build.gradle]: Ok :+1:

**Comment by iamareebjamal** [comment]
@niranjan94 Please approve

**Pull request merged**